apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/RetryTiming.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at 
+//
+// https://www.apache.org/licenses/LICENSE-2.0 
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and 
+// limitations under the License.
+
+using System;
+using System.Threading.Tasks;
+
+namespace Google.Cloud.Storage.V1;
+
+/// <summary>
+/// Options to control the delays between attempts when retrying failed API requests. These are immutable.
+/// </summary>
+public sealed class RetryTiming
+{
+    /// <summary>
+    /// The backoff time between the first attempt and the first retry.
+    /// </summary>
+    public TimeSpan InitialBackoff { get; }
+
+    /// <summary>
+    /// Maximum backoff time between retries.
+    /// </summary>
+    public TimeSpan MaxBackoff { get; }
+
+    /// <summary>
+    /// The multiplier to apply to the backoff on each iteration.
+    /// </summary>
+    public double BackoffMultiplier { get; }
+
+    /// <summary>
+    /// The default initial backoff time between the first attempt and the first retry. The default is 1 second.
+    /// </summary>
+    public static TimeSpan DefaultInitialBackoff { get; } = TimeSpan.FromSeconds(1);
+
+    /// <summary>
+    /// The default maximum backoff time between retries. The default is 32 seconds.
+    /// </summary>
+    public static TimeSpan DefaultMaxBackoff { get; } = TimeSpan.FromSeconds(32);
+
+    /// <summary>
+    /// The default maximum backoff multiplier to be applied on each iteration. The default is 2.
+    /// </summary>
+    public static double DefaultBackoffMultiplier { get; } = 2;
+
+    /// <summary>
+    /// Returns the retry timing configurations all set to default.
+    /// </summary>
+    public static RetryTiming Default { get; } = new();
+
+    /// <summary>
+    /// Returns the existing retry timings configurations using the specified initial backoff.
```

Comment:
```suggestion
    /// Returns a new instance with the same values as the current instance, but with the specified initial backoff.
```

(And the same for the other methods.)

I know that basically repeats the `<return>` element, but I think it's clearer than "existing".